### ISSUE TITLE
how to update

### ISSUE THREAD

**Issue by szxr** [Sxip/jam#8]
I know jam is discontinued now and I've tried to update it myself but no luck.
If you could just send some instructions on how to make it work with version 1621 I would really appreciate it 😊

**Comment by LYNCAT12** [comment]
hey! i found out how to use jam without updating it. simply use jam master and use animal jam browser

**Comment by szxr** [comment]
yeah id think id rather just update it instead but thanks also i think i already found the correct remote host which is 35.85.129.213 but its still not working D:
